### DISCUSSION TITLE
Extend 'SortingFn' API with direction

### DISCUSSION THREAD

**Discussion by martinboksa** [TanStack/table#4663] [Ideas]
Hi guys,
It would be very nice to extend sorting fn with direction.

## My use case:
My table contains 2-row types: 'sets' and 'items'. And I need to sort only 'items' which are children of 'sets'. It means 'sets' have always the same position and only their items change positions. This works perfectly

**PROBLEM:** Item has a special rule that 'undefined' value should be always at the bottom and I cannot do it without 'sort direction' in API

<img width="862" alt="image" src="https://user-images.githubusercontent.com/4490694/212871693-2122c35e-5e33-484e-a4cb-a227e2c2aa9b.png">

## How?
Pass parameter `isDesc` to function :)
https://github.com/TanStack/table/blob/4e3e735afac1d664545021f2e5c129080dcee7d9/packages/table-core/src/utils/getSortedRowModel.ts#L72

Thank you, guys! 🙏

**Comment by AndreaEsposit** [comment]
That would be a great idea :)

**Comment by Brookke** [comment]
I'd be happy to pick this up, if it were to be approved.

Currently we are using the following as a workaround: 

```
const direction = a
    .getAllCells()
    .find((cell) => cell.column.id === id)
    ?.column.getIsSorted();
```
